### PR TITLE
feat(kn): add plugin for `kn` completion

### PR DESCRIPTION
**Pull request by btannous**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- adding a plugin to add `kn` command line completion

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

**Comment by sboardwell** [comment]
Is there any reason this PR is not being included?

**Comment by btannous** [comment]
Not that I know of, maybe I didn't create an issue or follow the standard practice to get things merged? This is my first pr here :-)

**Comment by sboardwell** [comment]
@53jk1 - do you know if anything else is needed here?

**Comment by 53jk1** [comment]
> @53jk1 - do you know if anything else is needed here?

Everything seems okay

**Comment by jxlwqq** [comment]
Can this pr be merged?

**Comment by btannous** [comment]
Someone please do

File: plugins/kn/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# kn - Knative CLI
+
+This plugin provides autocompletion for [kn](https://knative.dev/docs/install/install-kn/) revison, route, service, source, and trigger operations.
```
Comment (mcornella) [inline plugins/kn/README.md:3]: This URL doesn't work anymore, can you fix that?

Comment (navidshaikh) [reply plugins/kn/README.md:3]: please link https://knative.dev/docs/install/client/#kn

**Pull request merged**